locale/en/get-involved/node-meetups

```diff
@@ -0,0 +1,56 @@
+# Node Meetups
+
+This is a list of node meetups. Please submit a PR if you'd like to add your local group!
+
+## Code of Conduct
+
+If any meetup does not have a CoC and/or is reported as an unsafe place, it will be removed from this list. 
+
+## Notes for adding meetups
+
+FORMAT
+- [Meetup](https://www.meetup.com/pdxnode/)
+- Frequency of meetups 
+- How to submit a talk? <<list here>>
+- Organizer names (if you wish to provide)
+- Organizers contact info (if you wish to provide)
+
+REQUIREMENTS
+- Please state in your PR if this meetup abides by CoC.
+- Link to CoC for verification.
+- If you do not currently have a CoC, update the meetup with CoC before submitting.
+- Submit your PR in alphabetical order.
+
+## Meetups
+
+### United States
+
+#### California
+
+##### San Francisco
+
+- [Meetup](https://www.meetup.com/sfnode/)
+- Frequency of meetups - monthly
+- How to submit a talk? 
```

Comment:
Maybe?
```suggestion
- How to submit a talk? DM @dshaw on twitter
```

Reply:
@vsemozhetbyt leaving that for now until they want to update